### PR TITLE
Update NativeScript Windows Setup Installer Article

### PR DESCRIPTION
**Pull request by vchimev**

### CODE REVIEW

File: start/ns-setup-installer.md [hunk 1]
```diff
@@ -24,95 +24,71 @@ The installation requires .NET Framework 4 and Windows 7 SP1 or later.
...
 #### Node.js
 
-Shows whether a supported Node.js version is installed on the machine.
-
-![](../img/start/installer-011.png "Node.js version")
-
-If no such version detected an information about what version will be installed and how much download and installation space it requires is shown.
-
-If supported Node.js is installed but its NPM feature is not installed or not added to PATH environment variable then an appropriate message is shown and the &quot;Install&quot; button is disabled
+Shows whether a supported Node.js version is installed on the machine. If no such version detected, information about what version will be installed and how much space it requires is shown.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: If no such version is detected

File: start/ns-setup-installer.md [hunk 2]
```diff
@@ -24,95 +24,71 @@ The installation requires .NET Framework 4 and Windows 7 SP1 or later.
...
 
-If a version higher than the supported is detected – a warning message is shown.
+If supported Node.js is installed but its NPM feature is not installed or not added to PATH environment variable then, an appropriate message is shown and the &quot;Install&quot; button is disabled.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: If a supported version of Node.js is installed, but its npm feature is not, or if the npm command has not been added to the PATH environment variable, an appropriate message is...

File: start/ns-setup-installer.md [hunk 3]
```diff
@@ -24,95 +24,71 @@ The installation requires .NET Framework 4 and Windows 7 SP1 or later.
...
 
 #### Android SDK
 
-Shows whether Android SDK is installed on the machine.
-
-![](../img/start/installer-015.png "Android SDK version")
-
-If an Android SDK is not detected, information about what version will be installed and how much download and installation space it requires is shown.
+Shows whether Android SDK is installed on the machine. If an Android SDK is not detected, information about what version will be installed and how much space it requires is shown.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: Shows whether an Android SDK is...

**Review by NickIliev — APPROVED** [review]

File: start/ns-setup-installer.md [hunk 4]
```diff
@@ -24,95 +24,71 @@ The installation requires .NET Framework 4 and Windows 7 SP1 or later.
...
+- &quot;Update SDK features&quot; – always visible. Use this option to update the installed SDK features
 
-### Installation Page
+![](../img/start/installer-007.png "Start Installation Popup")
 
-Shows a progress bar and information about the current installation/setup that is executing. All buttons are disabled. The operation cannot be canceled. The installer cannot be closed and the &quot;X&quot; button is disabled, too.
+_**Install**_ button
 
-![](../img/start/installer-017.png "Installation Page")
+A dialog is shown to the user to confirm the installation as it can take a lot of time and cannot be canceled. If &quot;Yes&quot; is clicked proceeds to the installation page and the installation process is started. If &quot;No&quot; is clicked the dialog is closed.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: If Yes is clicked, it proceeds...

File: start/ns-setup-installer.md [hunk 5]
```diff
@@ -24,95 +24,71 @@ The installation requires .NET Framework 4 and Windows 7 SP1 or later.
...
-#### Success
-![](../img/start/installer-021.png "Success")
+![](../img/start/installer-017.png "Installation Screen")
 
-If the &quot;Open installation log&quot; checkbox is checked when &quot;Finish&quot; is clicked – the detailed installation log is opened. In case the installer ends up with  an error summary page then you need to go through the advanced setup for Windows and manually install of the prerequisites.
+### Installation successful
 
+![](../img/start/installer-021.png "Installation successful")
 
+After successful installation, the installer shows a reference to a [documentation article](https://docs.nativescript.org/tooling/android-virtual-devices) on how to setup Android Emulator. Also, a checkbox to open documentation is checked.
```
Comment (tjvantoll) [inline start/ns-setup-installer.md]: on how to set up an Android Emulator.

**Review by tjvantoll — COMMENTED** [review]
Couple of minor grammatical notes. Looks good.

File: start/ns-setup-installer.md (hunk 1 above)
Comment (vchimev) [reply start/ns-setup-installer.md]: Done, thanks.

File: start/ns-setup-installer.md (hunk 2 above)
Comment (vchimev) [reply start/ns-setup-installer.md]: Done, thanks.

File: start/ns-setup-installer.md (hunk 3 above)
Comment (vchimev) [reply start/ns-setup-installer.md]: Done, thanks.

File: start/ns-setup-installer.md (hunk 4 above)
Comment (vchimev) [reply start/ns-setup-installer.md]: Done, thanks.

File: start/ns-setup-installer.md (hunk 5 above)
Comment (vchimev) [reply start/ns-setup-installer.md]: Done, thanks.

**Comment by tjvantoll** [comment]
Looks good 👍

**Pull request merged**